### ISSUE TITLE
feat(scope): Action History Functionality

### ISSUE THREAD

**Issue by kat0d** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#657]
### Describe the feature you want

Please, add action history, like, when I uninstall/disable package, or restore/enable.
Because there is a situation. Like, I had remove some uninstall packages from recomended on Xiaomi android 14. And my mobile internet stop working. It was impossible to add APN Point ("Access Point Name settings are not available for this user"). I had enable/restore few packages, and it's all fine.
And the point is, that I can't remember, what packages I had restore. And if there was a history, then I could determine which package should not be removed

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by AnonymousWP** [comment]
> ### Describe the feature you want
> Please, add action history, like, when I uninstall/disable package, or restore/enable. Because there is a situation. Like, I had remove some uninstall packages from recomended on Xiaomi android 14. And my mobile internet stop working. It was impossible to add APN Point ("Access Point Name settings are not available for this user"). I had enable/restore few packages, and it's all fine. And the point is, that I can't remember, what packages I had restore. And if there was a history, then I could determine which package should not be removed
> 
> ### Acknowledgements
> * [x]  This issue is not a duplicate of an existing feature request.
> * [x]  I have chosen an appropriate title.
> * [x]  All requested information has been provided properly.

There are logs for that: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ#where-are-the-logfiles

**Comment by Rudxain** [comment]
I agree that the logs should be enough, as they can be searched/`grep`ed. Unless they meant something like _"undo/redo"._

Some file managers support undoing reversible actions, such as moving/renaming files, but not undoing a deletion (unless the files were sent to the recycle-bin)
